django_cas_ng/backends.py

```diff
@@ -26,11 +26,15 @@ def authenticate(self, ticket, service, request):
         if not username:
             return None
 
-        username_case = settings.CAS_FORCE_CHANGE_USERNAME_CASE
-        if username_case == 'lower':
-            username = username.lower()
-        elif username_case == 'upper':
-            username = username.upper()
+        for func in settings.CAS_USERNAME_NORMALIZATIONS:
+            if callable(func):
+                username = func(username)
+            elif func == 'lower':
+                username = username.lower()
+            elif func == 'upper':
+                username = username.upper()
+            elif func == 'strip':
+                username = username.strip()
 
```

Comment:
should add else clause to indicate there is an error in config.
error should not be ignore silently.